### PR TITLE
CY-144 - Update documentation (Stage Help)

### PR DESCRIPTION
**Pull request by qooban**
Changes:
* in Widget Structure section added `readme.md` file,
* in Widget Settings table added new fields - `hasReadme` and `helpUrl`.

This PR bases on #689 => #689 shall be merged first to `master`, then `next` shall be updated from `master` and after that this PR can be merged.

### CODE REVIEW

File: content/manager_webui/custom-widgets.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ A widget is made up of these files:
 * `backend.js` - A widget backend that allows widget frontend code to use [backend services]({{< relref "manager_webui/custom-widgets.md#widget-backend" >}}) (**Optional**)
 * `widget.html` ‑ A widget template file that is relevant only when you are writing a widget using plain JavaScript with an HTML template (**Optional**)
 * `widget.css` ‑ The CSS file that the widget uses (**Optional**)
+* `readme.md` - A widget help ReadMe file in [Markdown format](https://en.wikipedia.org/wiki/Markdown) (**Optional**)
```
Comment (edenpessach) [inline content/manager_webui/custom-widgets.md]: README.md

File: content/manager_webui/custom-widgets.md [hunk 2]
```diff
@@ -41,6 +42,7 @@ Using this method, the file system will look as follows:
       widget.html
       widget.png
       widget.css
+      readme.md
```
Comment (edenpessach) [inline content/manager_webui/custom-widgets.md]: README.md

File: content/manager_webui/custom-widgets.md [hunk 3]
```diff
@@ -52,6 +54,7 @@ If you want to use **plain JavaScript** and you want to create a widget named `b
...
```
Comment (edenpessach) [inline content/manager_webui/custom-widgets.md]: README.md

**Comment by qooban** [comment]
@bmansheim Updated after yours review.

**Pull request merged**